src/styleguide/routes/directives/r-focus-mask.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div v-focus-mask :class="b()">
+    <h1>Focus mask</h1>
+    <h2>Default</h2>
+    <div v-focus-item="hasFocus">
+      I have the focus.
+      <br>
+      <br>
+      <button type="button"
```

Comment:
question: I expect that the trigger button could be placed outside the `v-focus-item` container.

When I move the `button` outside and toggle the mask, the trigger button gets covered by the mask. What are your thoughts on this behavior?

Reply:
This is correct. You need to place the related button inside the mask or outside of it's range. It's on purpose, that you can't interact with the things behind the mask.